app/commands/donations/payment_intent/create.rb

```diff
@@ -9,6 +9,11 @@ class Create
       initialize_with :user_or_email, :type, :amount_in_cents
 
       def call
+        if invalid_user_or_email?
+          Bugsnag.notify(PaymentIntentError.new("Invalid user or email trying to make donation: #{user_or_email}"))
```

Comment:
I noted that this class defined a custom `PaymentIntentError` class, so I thought: why not use that? (we also don't need the `begin/rescue` bit)

Reply:
Nice.